crates/time_format/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "time_format"
+version = "0.1.0"
+edition = "2021"
+publish = false
+license = "GPL-3.0-or-later"
+
+[lib]
+path = "src/time_format.rs"
+doctest = false
+
+[dependencies]
+anyhow.workspace = true
+lazy_static.workspace = true
+sys-locale.workspace = true
+time.workspace = true
+util.workspace = true
+
+[target.'cfg(target_os = "macos")'.dependencies]
+core-foundation = "0.9.4"
```

Comment:
core-foundation and core-foundation-sys are used in a lot of crates, can we should we move those dependencies to the workspace Cargo.toml?

Reply:
yes please! In general it's best to have dependencies listed there to avoid accidentally having two versions in flight.

Reply:
Did that, also removed the `with-uuid` feature, which was referenced in gpui. It does not seem to be used anywhere.